yb-voyager/cmd/exportDataFromTarget.go

```diff
@@ -108,3 +111,43 @@ func initSourceConfFromTargetConf() error {
 	source.Uri = targetConf.Uri
 	return nil
 }
+
+func packAndSendExportDataFromTargetPayload(status string) {
+	if !callhome.SendDiagnostics {
+		return
+	}
+	payload := createCallhomePayload()
+	switch exporterRole {
+	case TARGET_DB_EXPORTER_FF_ROLE:
+		payload.MigrationType = LIVE_MIGRATION_WITH_FALL_FORWARD
+	case TARGET_DB_EXPORTER_FB_ROLE:
+		payload.MigrationType = LIVE_MIGRATION_WITH_FALLBACK
+	}
+
+	targetDBDetails := callhome.TargetDBDetails{
+		Host:      source.Host,
+		DBVersion: source.DBVersion,
+	}
+	payload.SourceDBDetails = callhome.MarshalledJsonString(targetDBDetails)
```

Comment:
we should be careful about the terminology here. I'm not sure it's okay to call target details as source details (esp because our end users here are callhome users)

Reply:
No I think thats a miss in copy paste error from export data from source, I thought of putting it in targetdb details but by mistakenly kept it in sourcedb details.